.github/workflows/linux.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       with:
         fetch-depth: 2
 
+    - name: Check whitespace
+      uses: ./.github/actions/whitespace
+
+    - name: Check final newline
+      uses: qwrtln/check-final-newline@v1.0.0
+
     - name: Lint the code
       uses: ./.github/actions/lint
```

Comment:
Instead of adding these as new actions, can't they be integrated into the `./github/actions/lint`?

Reply:
Yep, good idea. Done.